frontend/src/metabase/auth/components/PasswordPanel/PasswordPanel.unit.spec.tsx

```diff
@@ -1,43 +1,39 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render, screen, waitFor } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { AuthProvider } from "metabase/auth/types";
-import PasswordPanel from "./PasswordPanel";
+import PasswordPanel, { PasswordPanelProps } from "./PasswordPanel";
 
 describe("PasswordPanel", () => {
   it("should login successfully", () => {
-    const onLogin = jest.fn().mockResolvedValue({});
+    const props = getProps();
+    const data = { username: "user@example.test", password: "password" };
 
-    render(<PasswordPanel onLogin={onLogin} />);
+    render(<PasswordPanel {...props} />);
+    userEvent.type(screen.getByLabelText("Email address"), data.username);
```

Comment:
You can easily write unit tests for forms now, no setup or imports mocking is required